test/sa_db_users.sql

```diff
@@ -69,6 +69,8 @@ GRANT SELECT ON certificates TO 'cert_checker'@'localhost';
 
 -- Expired authorization purger
 GRANT SELECT,DELETE ON pendingAuthorizations TO 'purger'@'localhost';
+GRANT SELECT,DELETE ON authz TO 'purger'@'localhost';
+GRANT SELECT,DELETE ON challenges TO 'purger'@'localhost';
```

Comment:
Can you open an ops ticket for these permission modifications ahead of this being merged? The last time I added some user `GRANT`s for a tool I forgot and broke a deploy :angel: